lms/static/js/certificates/views/certificate_bulk_whitelist.js

```diff
@@ -73,12 +74,91 @@
                 },
 
                 display_response: function(data_from_server) {
+                    var UserOnCertificateInvalidationList;
+
                     $('.bulk-exception-results').removeClass('hidden').empty();
 
+                    function generateDiv(group, heading, displayData) {
+                        // inner function generate div and display response messages.
+                        $('<div/>', {
+                            class: 'message ' + group
+                        }).appendTo('.bulk-exception-results').prepend( // eslint-disable-line max-len, xss-lint: disable=javascript-jquery-insert-into-target,javascript-jquery-prepend
+                                "<button type='button' id= '" + group + "' class='arrow'> + </button>" + heading) // eslint-disable-line max-len, xss-lint: disable=javascript-concat-html
+                        .append($('<ul/>', {
+                            class: group
+                        }));
+
+                        for (var i = 0; i < displayData.length; i++) { // eslint-disable-line vars-on-top
+                            $('<li/>', {
+                                text: displayData[i]
+                            }).appendTo('div.message > .' + group); // eslint-disable-line max-len, xss-lint: disable=javascript-jquery-insert-into-target
+                        }
+                        $('div.message > .' + group).hide();
+                    }
+
+                    function getDisplayText(qty, group) {
```

Comment:
Quality violation led me to change this from `get_text` to `getDisplayText`. We already are passing in a function (from backbone?) named `gettext` and I figured having both `getText` and `gettext` would be terrible.